diff --git a/build_tools/wheel_utils/Dockerfile.x86 b/build_tools/wheel_utils/Dockerfile.x86
--- a/build_tools/wheel_utils/Dockerfile.x86
+++ b/build_tools/wheel_utils/Dockerfile.x86
@@ -30,5 +30,5 @@
 RUN rm -rf /var/cache/dnf/*
 RUN echo "/usr/local/cuda/lib64" >> /etc/ld.so.conf.d/999_nvidia_cuda.conf
-RUN dnf -y install cuda-toolkit-${CUDA_MAJOR}
+RUN dnf -y install cuda-toolkit-${CUDA_MAJOR}-${CUDA_MINOR}
 RUN dnf clean all
 RUN dnf -y install glog.x86_64 glog-devel.x86_64
diff --git a/build_tools/wheel_utils/Dockerfile.aarch b/build_tools/wheel_utils/Dockerfile.aarch
--- a/build_tools/wheel_utils/Dockerfile.aarch
+++ b/build_tools/wheel_utils/Dockerfile.aarch
@@ -30,5 +30,5 @@
 RUN rm -rf /var/cache/dnf/*
 RUN echo "/usr/local/cuda/lib64" >> /etc/ld.so.conf.d/999_nvidia_cuda.conf
-RUN dnf -y install cuda-toolkit-${CUDA_MAJOR}
+RUN dnf -y install cuda-toolkit-${CUDA_MAJOR}-${CUDA_MINOR}
 RUN dnf clean all
 RUN dnf -y install glog.aarch64 glog-devel.aarch64
